### ISSUE TITLE
MSI installer of version 22.0.0 "hanging"

### ISSUE THREAD

**Issue by SkyBeam** [jgraph/drawio-desktop#1477]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

**Describe the bug**

**To Reproduce**
msiexec /i draw.io-22.0.0.msi /qn ALLUSERS=2 MSIINSTALLPERUSER=0
msiexec /x draw.io-22.0.0.msi /qn

**Expected behavior**
Package to be installed and uninstalled silently.

**Screenshots**
Unfortunately there is no output, the process simply "hanging"

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 22.0.0

**Desktop (please complete the following information):**
 - OS: Windows (for sure on Windows 11 22H2 Build 22621.2361)

**Additional context**
I am facing this problem when upgrading from version 21.8.2 but could also experience the MSI process to hang when uninstalling even if a clean installation is performed.

**Comment by davidjgraph** [comment]
This report isn't precise enough, it doesn't contain full environment details.

**Comment by SkyBeam** [comment]
Not sure what else you need. Happening on basically any Windows 10 and Windows 11 I tested (all latest updates from 2023-09 cumulative update installed on version 22H2).

**Comment by SkyBeam** [comment]
I have to apologize for wasting your and my time. Just now I re-tested the same drawio 22.0.0 MSI installer (did not re-download either) and it worked fine. Also uninstalling did not hang any more.
I do not h ave an explanation for this right now other than potential impact of Windows Defender or similar side-effects which got resolved. Also recently faced issues about Windows Defender silently dropping tor.exe from Tor Browser essentially silently breaking it. I did however not find anything meaningful in the logs about draw.io Desktop yet.

So one day after having the issue on multiple fully updated Windows 10 and 11 22H2 I am unable to reproduce the issue on any of those. Therefore I am closing the issue and stop wasting your time.

Thank you for responding to the report anyway!
